src/container.js

```diff
@@ -45,7 +48,7 @@ export default ({
   logger = console
 } = {}) => {
   const container = {
-    pinia: createPinia(),
+    pinia: process.env.NODE_ENV === 'development' ? createPinia() : piniaMock,
```

Comment:
Originally, my strategy was going to be to use webpack to alias the `pinia` package to src/util/pinia.js. That way, only our webpack config would change, not this file. However, at the end of the day, I think this approach is clearer: there's less webpack magic involved. Plus, now that we're planning to move to Vite (#671), it seems good to have as little webpack-specific code as possible. I checked the build, and even though this file still imports Pinia, if it doesn't use Pinia at all, which it won't outside development, the import is removed from the bundle.

Since we won't use it after all, I've removed src/util/pinia.js.